sale_order_line_mrp_remarks/models/sale_order.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models, api
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    @api.multi
+    def action_confirm(self):
```

Comment:
@yostashiro, can you review this method?

Reply:
@TimLai125 I was thinking that we should hook the logic in create method of mrp.production instead.  I suppose we should be able to find the related sales order through procurement_group_id. (I think we talked about using this field)

Reply:
In case there are two manufacturing products in the same sales order, all the manufacturing orders will have the same procurement_group_id. Therefore it is not possible to pass the correct remarks field to the MO in this pattern.

Reply:
>> we should hook the logic in create method of mrp.production instead

Let me check this approach

Reply:
@yostashiro I have checked the "hook the logic in create method of mrp.production" approach. Since only the first creation of MO is directly triggered by specific sales order line, other MOs creations are triggered by `stock.move` records. Extra logic would be needed to keep the remarks field in the stock.move if we do it in that way. Consider that approach will not be easily understood by others in the future, I have further updated this method that it should handle all the situation that we have discussed, i.e. https://github.com/rfhk/cip-custom/pull/122/commits/17d0fbe2ee5ce4d05d54608e6f9327edd9b49959